Release Notes/2019/2019-04-02-cloud-platform-release-notes.md

```diff
@@ -23,20 +23,19 @@ Cloud Application Manager now supports Lumen Master Account as an authentication
 
-
 ##### Support for multiple ServiceNow instances
 
-Cloud Application Manager now adds support to configure multiple ServiceNow instances to integrate with. The organization settings page provides the ability to set up multiple ServiceNow instance configurations within the organization. From there, every deployment of an instance in Cloud Application Manager will inform all ServiceNow instances CMDB about the new resource, will show its IDs in the instance details page, and will inform all ServiceNow instances CMDB about termination of an instance so it can be removed there as well.
+Cloud Application Manager now adds support to configure multiple ServiceNow instances to integrate with. The organization settings page provides the ability to set up multiple ServiceNow instance configurations within the organization. From there, every deployment of an instance in Cloud Application Manager will inform all ServiceNow instances in the configuration management database (CMDB) about the new resource, will show its IDs in the instance details page, and will inform all ServiceNow instances in the CMDB about termination of an instance so it can be removed there as well.
 
 ##### User Interface Redesign for Provider Services
 
-Cloud Application Manager has redesigned the user interface formerly used to enable services such as [Managed Services Anywhere](../../Cloud Application Manager/Managed Services/getting-started-with-cam-enable-managed-provider.md), [Automatic Discovery of Resources](../../Cloud Application Manager/Getting Started/register-existing-instance.md) and [Analytics](../../Cloud Application Manager/analytics/cloudapplicationmanageranalyticsui.md). The services toggles that were available in the Edit Provider dialog has been moved to a new tab called Services in the provider details page, where you can manage them and see additional details such as the status of your Managed Services Anywhere enabled locations.
+Cloud Application Manager has redesigned the user interface formerly used to enable services such as [Managed Services Anywhere](https://www.ctl.io/knowledge-base/cloud-application-manager/managed-services/getting-started-with-cam-enable-managed-provider/), [Automatic Discovery of Resources](https://www.ctl.io/knowledge-base/cloud-application-manager/getting-started/register-existing-instance/) and [Analytics](https://www.ctl.io/knowledge-base/cloud-application-manager/analytics/cloudapplicationmanageranalyticsui/). The services toggles that were available in the Edit Provider dialog has been moved to a new tab called Services in the provider details page, where you can manage them and see additional details such as the status of your Managed Services Anywhere enabled locations.
```

Comment:
Is there are reason that you are using absolute links instead of the relative links that were there?